README.md

```diff
@@ -31,23 +30,26 @@ no cached entry is taken. No message is printed to the logfile.
 
 ## Compressed Content Exchange Mechanism (CCEM)
```

Comment:
Please provide some link to the documentation of this feature, if it's an LDAP feature.

Reply:
No idea what it is and out of the scope of this PR